data/json/furniture_and_terrain/appliances.json

```diff
@@ -575,6 +575,20 @@
     "copy-from": "dishwasher",
     "item": "household_dishwasher"
   },
+  {
+    "type": "vehicle_part",
+    "id": "ap_autoclave",
+    "name": { "str": "autoclave" },
```

Comment:
Maybe call this a `small autoclave` of a `20L autoclave` to differentiate it from the big furniture ones in labs.